test/unit/test_gut_yielding.gd

```diff
@@ -28,8 +28,13 @@ class TimedSignaler:
 class TimeSignalerParam:
 	extends TimedSignaler
 
+	var _emitted_param
+
+	func _init(emitted_param = 1):
+		_emittied_param = emitted_param
+
 	func _on_timer_timeout():
-		emit_signal('the_signal', 1)
+		emit_signal('the_signal', emitted_param)
```

Comment:
The variable and parameter names are all wrong here. ` _on_timer_timeout` doesn't use the property, `_init` isn't assigning to the right thing.

Reply:
:eyeglasses: